cli/tests/cli_tests.rs

```diff
@@ -70,11 +70,10 @@ function fail() {
         &std::env::current_dir().context("unable to get current working directory")?,
     )?;
 
+    // FIXME: stderr should include the source code line constructing the error
     let expected_stderr = format!(
         r#"
-  error: Uncaught Error: boom!
-  throw new Error("boom!");
-        ^
+  error: Uncaught (in promise) Error: boom!
```

Comment:
This is a regression I'd like to fix, but it's not blocking this PR from landing.

Reply:
Did you open a follow up tech debt issue?

Reply:
looks like this was fixed in https://github.com/CheckerNetwork/zinnia/pull/717/commits/1874fc8dc173291de6ecd113b229070911ff84e0 and this conversation can be resolved